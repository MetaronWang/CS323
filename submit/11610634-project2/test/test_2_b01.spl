bool equal(int a, int b){
    return a==b;
}

int add(int a, int b){
    return a+b;
}

int main(){
    if(equal(1,2)){
        int a;
    }
    if(add(1,2)){
        int b;
    }
    return 0;
}