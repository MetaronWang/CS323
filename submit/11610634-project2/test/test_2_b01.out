Error type 18 at Line 13: Only Bool can be used in the if()/while()
